test/runtime/configMatters/mem/align.chpl

```diff
@@ -4,15 +4,15 @@ use CTypes;
 // try with size = 2*alignment
 for i in 3..20 {
   const alignment = (1 << i) : uint;
-  var ptr = c_aligned_alloc(uint(8), alignment, 2*alignment);
+  var ptr = allocate(uint(8), 2*alignment, alignment=alignment);
```

Comment:
Mostly noticing on this test, but I think others are affected. In some cases you have an explicit cast to `c_size_t` and some places you don't. I am assuming the places you have it are required for the code to work correctly?

Reply:
And after reviewing the code again, I seethe cast in places where the value is signed (ie `int(64)`). So that answers my question

Reply:
My only logic for where I added the cast was places I got warnings, which I think yes was sized ints. Now that I think about it, I wonder if 32 bit systems will run into additional warnings I haven't, since I think `c_size_t` would be smaller there... I don't have any on hand to test that with though, so may just let nightly catch those cases if they exist

Reply:
> may just let nightly catch those cases if they exist

Wow, this came back to bite me really hard